core-tests/shared/src/test/scala/zio/stm/TMapSpec.scala

```diff
@@ -25,53 +25,55 @@ object TMapSpec extends ZIOBaseSpec {
   def spec = suite("TMap")(
     suite("factories")(
       testM("apply") {
-        val tx = TMap.make("a" -> 1, "b" -> 2, "c" -> 2, "b" -> 3).flatMap(_.toList)
+        val tx = TMap.make("a" -> 1, "b" -> 2, "c" -> 2, "b" -> 3).flatMap[Any, Nothing, List[(String, Int)]](_.toList)
```

Comment:
This is unfortunate :(

Reply:
This is our good friend https://github.com/scala/bug/issues/11798 showing its ugly head

Reply:
The fix is in again for that lint limitation.